pkgs/development/python-modules/torch/default.nix

```diff
@@ -143,12 +143,10 @@ in buildPythonPackage rec {
     # base is 10.12. Until we upgrade, we can fall back on the older
     # pthread support.
     ./pthreadpool-disable-gcd.diff
-  ] ++ lib.optionals stdenv.isLinux [
+  ] ++ lib.optionals (stdenv.isLinux && (cudaSupport || rocmSupport)) [
```

Comment:
I don't think conditional patches are a good idea, they just add to the complexity

Reply:
Fixed in force-push. Will update the PR description.